test-loop-tests/src/tests/catching_up.rs

```diff
@@ -375,27 +371,11 @@ fn ultra_slow_test_catchup_sanity_blocks_produced() {
 }
 
 #[test]
-fn ultra_slow_test_all_chunks_accepted_1000() {
-    test_all_chunks_accepted_common(1000, 3000, 7)
-}
-
-#[test]
-fn ultra_slow_test_all_chunks_accepted_1000_slow() {
-    test_all_chunks_accepted_common(1000, 6000, 7)
-}
-
-#[test]
-fn ultra_slow_test_all_chunks_accepted_1000_rare_epoch_changing() {
-    test_all_chunks_accepted_common(1000, 1500, 100)
-}
-
-fn test_all_chunks_accepted_common(
-    last_height: BlockHeight,
-    block_prod_time: u64,
-    epoch_length: BlockHeightDelta,
-) {
+fn slow_test_all_chunks_accepted() {
     init_test_logger();
 
+    let epoch_length = 20;
```

Comment:
Increased epoch length from 7 to 20 as that was the cause of the missing chunks.